sidebars.js

```diff
@@ -701,6 +701,7 @@ module.exports = {
             "self-managed/zeebe-deployment/operations/disk-space",
             "self-managed/zeebe-deployment/operations/update-zeebe",
             "self-managed/zeebe-deployment/operations/rebalancing",
+            "self-managed/zeebe-deployment/operations/management-api",
```

Comment:
Due to the fact that we now have separate versions for Optimize than the rest of the docs, parallel sidebar changes will need to be made at https://github.com/camunda/camunda-platform-docs/blob/main/optimize_sidebars.js#L1312.

Reply:
Thanks. I have updated it.